package.sh

```diff
@@ -4,20 +4,69 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
+
+FRAMEWORK="drupal"
+INSTALL_DOCKER=false
+RED='\033[31m'
+GREEN='\033[32m'
+YELLOW='\033[33m'
+RESET='\033[0m'
+OS=$(uname)
+USER=$(whoami)
+
+# @todo Checks framework argument.
+ARGS=$(getopt -o hdf: -l "help,docker,framework:" --name "$0" -- "$@")
+
+eval set -- "$ARGS"
+
+while [ $# -gt 0 ]; do
+    case "$1" in
+        -f|--framework)
+            shift;
+            if [ -n "$1" ]; then
+                FRAMEWORK="$1"
+                if [ ! -d "${PWD}/framework/${FRAMEWORK}" ]; then
+                    printf "${RED}Framework ${FRAMEWORK} directory not found.\n${RESET}"
+                    exit
+                fi
+                shift;
+            fi
+            ;;
+        -d|--docker)
+            INSTALL_DOCKER=true
+            shift;
+            ;;
+        -h|--help)
+            echo "Usage: package.sh [--help] [--framework drupal]"
+            echo ""
+            echo "Packages a contribution kit for the specified framework."
+            echo ""
+            echo "Arguments:"
+            echo "      --framework NAME ${GREEN}Builds the package for the specified framework (Drupal).${RESET}"
+            echo "      --docker         ${GREEN}Includes Docker installers in the package.${RESET}"
+            echo "      --help           ${GREEN}Prints this message and exits.${RESET}"
+            exit
+            shift;
+            ;;
+        --)
+            shift;
+            break
+            ;;
+    esac
```

Comment:
This probably needs a bit of hardening with a default case option so that it doesn't infinite loop. I added something like this to another bash script earlier ago:

```
*)
   printf "${YELLOW}#### WARNING: Ignoring unknown argument ${1}\n${RESET}"
   shift;
   break
   ;;
```